src/init.cpp

```diff
@@ -624,9 +624,23 @@ void ThreadStartWalletNotifier()
 
         if (walletBestBlockHash.has_value()) {
             int64_t slept{0};
+            bool isReindexing{true};
             auto timedOut = [&]() -> bool {
-                MilliSleep(50);
-                slept += 50;
+                MilliSleep(500);
+                {
+                    LOCK(cs_main);
```

Comment:
```suggestion
```
This lock is unnecessary.

Reply:
`blocktree` doesn't need to be guarded by a lock?